packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -481,17 +412,17 @@ void _applyCameraUpdate(gmaps.GMap map, CameraUpdate update) {
 
 // original JS by: Byron Singh (https://stackoverflow.com/a/30541162)
 gmaps.LatLng _pixelToLatLng(gmaps.GMap map, int x, int y) {
-  final ne = map.bounds.northEast;
-  final sw = map.bounds.southWest;
-  final projection = map.projection;
+  final ne = map.bounds!.northEast;
+  final sw = map.bounds!.southWest;
+  final projection = map.projection!;
```

Comment:
Why is all of this safe?

Reply:
I added a bunch of asserts to make it safe, or throwy-with-an-explanation.